client/src/components/statistics/Statistics.js

```diff
@@ -0,0 +1,25 @@
+import SeriesChart from "./SeriesChart";
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+const Statistics = () => {
+    const [series, setRecommendedSeries] = useState([]);
+
+    useEffect(() => {
+      async function fetchData() {
```

Comment:
Same comments as the other PR - config file, separate service file for HTTP requests, real API usage after Shuker's merge to main